zeekpkg/_util.py

```diff
@@ -69,37 +86,83 @@ def make_symlink(target_path, link_path, force=True):
             raise error
 
 
-def safe_tarfile_extractall(tfile, destdir):
-    """Wrapper to tarfile.extractall(), checking for path traversal.
+def zkg_tarfile_create(basedir):
```

Comment:
Could you add [PEP257 docstrings](https://peps.python.org/pep-0257/) for the functions you are adding?